README.md

```diff
@@ -315,25 +315,27 @@ import DefaultTheme from 'react-dates/lib/theme/DefaultTheme';
 
 ThemedStyleSheet.registerInterface(aphroditeInterface);
 ThemedStyleSheet.registerTheme({
-  ...DefaultTheme,
-  color: {
-    ...DefaultTheme.color,
-    highlighted: {
-      backgroundColor: '#82E0AA',
-      backgroundColor_active: '#58D68D',
-      backgroundColor_hover: '#58D68D',
-      color: '#186A3B',
-      color_active: '#186A3B',
-      color_hover: '#186A3B',
+  reactDates: {
+    ...DefaultTheme.reactDates,
+    color: {
+      ...DefaultTheme.color,
```

Comment:
Does this not need to be `DefaultTheme.reactDates.color` as well? 

Also thanks for the catch! :) 

@lencioni does theme registration work by overriding everything or by merging? I guess it wouldn't really matter since this theme is namespaced anyways....

Reply:
You're totally right! I just pushed the change. I have that in our app's code as well so not sure why I didn't commit it like that 🤔

Reply:
@majapw It simply overrides everything See here: https://github.com/airbnb/react-with-styles/blob/e17dea580ed45b5d8a4d7974354e94e8b6706671/src/ThemedStyleSheet.js#L8-L13